cli/command/environment.go

```diff
@@ -269,21 +288,31 @@ func (e *EnvironmentCommand) unlink(c *cli.Context) error {
 	}
 
 	if sourceEnvironmentID == destEnvironmentID {
-		return fmt.Errorf("Cannot unlink an environment from itself")
+		return fmt.Errorf("Cannot link an environment to itself")
 	}
 
-	req := models.DeleteEnvironmentLinkRequest{
-		SourceEnvironmentID: sourceEnvironmentID,
-		DestEnvironmentID:   destEnvironmentID,
-	}
+	addLinkFN := func(sourceEnvID, destEnvID string) error {
```

Comment:
a more appropriate name would probably be something like `updateLinkFN`, given that this could add or remove links

Reply:
Agreed. ✅